src/main/java/nl/tudelft/pixelperfect/client/EventsMessage.java

```diff
@@ -0,0 +1,82 @@
+package nl.tudelft.pixelperfect.client;
+
+import com.jme3.network.AbstractMessage;
+import com.jme3.network.serializing.Serializable;
+
+
+/**
+ * Message type that sends the event to the client.
+ * 
+ * @author Dmitry Malarev
+ *
+ */
+@Serializable
+public class EventsMessage extends AbstractMessage {
+
+  public int id;
+  public String type;
+  public Long timestamp;
+  public Long duration;
+
+  /**
+   * Constructs a new EventsMessage instance.
+   * 
+   */
+  public EventsMessage() {
+  }
+
+  /**
+   * Changes the content of the message.
+   *
+   * @param identifier
+   *          The id of the event.
+   * @param type
+   *          The type of the event.
+   * @param started
+   *          The timestamp of the event.
+   * @param limit
+   *          The time limit of the event.
+   */
+  public EventsMessage(int identifier ,String type, Long started, Long limit) {

```

Comment:
Well done, nice way to send the Event's parameters. But why the different parameter names, instead of id, type, timestamp and duration?